src/main/java/com/passpnu/passwordmanager/entity/Password.java

```diff
@@ -0,0 +1,26 @@
+package com.passpnu.passwordmanager.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Password {
```

Comment:
add `Entity` suffix e.g. `PasswordEntity`

Reply:
Entity suffix is added for all entities in the package